CHANGELOG.md

```diff
@@ -2,3 +2,4 @@
 - Fixes an issue where `ext:list` would fail when no extensions were installed.
 - Fixes an issue where database emulator could not load rules from target configurations (#2184).
 - Adds new API commands that handle list/create/delete operations on the Android SHA certificate hashes `apps:android:sha:list`, `apps:android:sha:create`, and `apps:android:sha:delete`.
+- Fixes an issue where the CLI did not send the proper auth tokens when performing Firestore / RTDB emulator operations.
```

Comment:
```suggestion
- Fixes an issue where the CLI did not assume admin privileges when performing Firestore / RTDB emulator operations.
```

How do you feel about this one? I'd like to mention `admin privileges` so people know this fixed their rules / permission denied issues.